code/auto_download/auto-download-lanl-covid19.r

```diff
@@ -9,12 +9,13 @@ library("dplyr")
 library("tidyr")
 library("readr")
 library("janitor")
+source(here("code", "config_utils", "get_forecast_targets.R"))
 
 model_name <- "LANL-GrowthRate"
 raw_dir <- file.path(tempdir(), "data-raw", model_name)
 processed_dir <- here::here("data-processed", model_name)
 last_sunday <- floor_date(today(), unit = "week", week_start = 7)
-data_types <- c("cases", "deaths")
+data_types <- get_forecast_targets()
```

Comment:
Would this make more sense as `get_hub_config("targets")` and `get_hub_config("ensemble_method")` so that we don't have to maintain a separate accessor function for every config option?